integration-tests/servlet/pom.xml

```diff
@@ -31,6 +31,11 @@
     <name>Camel Quarkus :: Integration Tests :: Servlet</name>
     <description>Integration tests for Camel Servlet component</description>
 
+    <properties>
+        <!-- Skip enforcer plugin as we want to use jboss marshalling for test class -->
+        <enforcer.skip>true</enforcer.skip>
```

Comment:
Same here: it was passing with the enforcer on, IIRC. Are there other reasons for disabling it?

Reply:
I found this comment on other poms so maybe it is not needed at all